lr_face/models.py

```diff
@@ -276,6 +278,10 @@ def get_model(self):
             module_name = f'insightface.{self.value}'
             module = importlib.import_module(module_name)
             return module.loadModel()
+        if self == self.KERAS_VGGFACE or self == self.KERAS_VGGFACE_RESNET:
+            module_name = f'keras_vggface.{self.value}'
+            module = importlib.import_module(module_name)
+            return module.load_model()
```

Comment:
hier is het load_model en hierboven loadModel. Ik vind load_model ook mooier, maar niet zo mooi als het door elkaar gebruikt wordt

Reply:
Done